course_inventory/inventory.py

```diff
@@ -14,18 +14,18 @@
 from db.db_creator import DBCreator
 from environ import ENV
 from vocab import ValidDataSourceName
-from .async_enroll_gatherer import AsyncEnrollGatherer
-from .canvas_course_usage import CanvasCourseUsage
-from .gql_queries import queries as QUERIES
-from .published_date import FetchPublishedDate
+from course_inventory.async_enroll_gatherer import AsyncEnrollGatherer
+from course_inventory.canvas_course_usage import CanvasCourseUsage
+from course_inventory.gql_queries import queries as QUERIES
+from course_inventory.published_date import FetchPublishedDate
 
 
 # Initialize settings and globals
 
 logger = logging.getLogger(__name__)
 
 ACCOUNT_ID = ENV.get('CANVAS_ACCOUNT_ID', 1)
-TERM_ID = ENV['CANVAS_TERM_ID']
+TERM_IDS = ENV['CANVAS_TERM_IDS']
```

Comment:
What verifies that `CANVAS_TERM_IDS` is definitely an array from the JSON and contains values?

Reply:
I guess nothing. This is a key configuration variable, as the application needs this input to direct what data to gather. I don't think there is a good default. What are you suggesting here?

Reply:
I suggest that the type of `ENV['CANVAS_TERM_IDS']` is verified to be a list.  (Data coming from the JSON file should be immutable, so maybe it should be a tuple instead.)  It should not be an empty list and its values should not be `None`.

Reply:
This is good, I think I lack experience in properly handling inputs. Can you tell me why it shouldn't be an empty array?

Reply:
The code needs at least one term to work with, doesn't it?  Have I misunderstood what it's doing with the term IDs?

Reply:
Yeah, I think you're right that it would probably break somewhere with no terms. I'm asking in part because of the `canvas_zoom_placements`, which can accept a null value and instead process specific course IDs. I'll think about this some more and commit a possible solution.

Reply:
You could check it, but if you don't you'll just get an exception somewhere, either a KeyError here or some other exception somewhere else. You could use `TERM_IDS = ENV.get("CANVAS_TERM_IDS", [])` here to avoid the KeyError but then you'd still have to 

```
if not isinstance(TERM_IDS, list):
   # Print some error message

```

I don't know *really* how important this is over just letting this throw an exception if this is incorrectly defined since it's just us using it though.

I also think if this became an issue we'd go down defining this as a Json Schema and validating that instead of doing this one off stuff. https://github.com/Julian/jsonschema.

Reply:
Yeah, I was kind of thinking along those lines, particularly because @lsloan could have brought up this issue with many of the variables in use here. As someone who has used schemas before with XML/Relax NG, I really like the idea of using JSON Schema to validate config files. Seems like it would be a useful tool for collaboration, also.

Reply:
@lsloan, do you think this needs to be resolved now, or can I create a separate issue, referencing the JSON Schema discussion?

Reply:
I don't think it's urgent enough to delay this PR.  Make it an issue.

Reply:
Thanks, I've created issue #119.